library/spdm_responder_lib/libspdm_rsp_measurements.c

```diff
@@ -83,7 +85,9 @@ void spdm_create_measurement_opaque(IN spdm_context_t *spdm_context,
 	ptr = (void *)((uint8_t *)response_message + response_message_size -
 		       measurment_no_sig_size);
 
-	spdm_get_random_number(SPDM_NONCE_SIZE, ptr);
+	if(!spdm_get_random_number(SPDM_NONCE_SIZE, ptr)) {
```

Comment:
It looks like spdm_create_measurement_opaque needs a return value, but we can do that in a separate pull request.